src/doc/unstable-book/src/library-features/allocator-api.md

```diff
@@ -2,6 +2,12 @@
 
 The tracking issue for this feature is [#32838]
 
+Items:
+
+- [core::alloc::Allocator](https://doc.rust-lang.org/nightly/core/alloc/trait.Allocator.html)
+- [alloc::alloc::Global](https://doc.rust-lang.org/nightly/alloc/alloc/struct.Global.html)
+- [core::alloc::AllocError](https://doc.rust-lang.org/nightly/core/alloc/struct.AllocError.html)
```

Comment:
~~Hmm, I thought we'd discussed automatically generating these so they don't get outdated?~~

never mind, I see that tidy suggests these automatically if they get outdated. But then why do some of the files below use `{items}` instead?

Reply:
It is because each feature does not have an explicit file, most of them are actually autogenerated in the `unstable-book-gen` tool.
I made it so that Tidy errors when an explicitly written feature misses an item and silently ignores the missing features.
In the `unstable-book-gen` tool, I do the same work as Tidy, but this time to include it in the autogenerated page.